CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+##Contributing to this repository
+
+Welcome!  Thanks for your interest in contributing to Software Carpentry's introductory shell lesson.  
+
+###Intro
+
+Software Carpentry is an open source project, and we welcome contributions of all kinds: new and improved lessons, bug reports, and small fixes to existing material are all useful.
+
+By contributing, you are agreeing that Software Carpentry may redistribute your work under these licenses.
+
+###Making and Submitting Changes
+
+Don't know where to start?  Please see the current list of issues (linked in the right sidebar) for ideas for contributing to this repository. 
```

Comment:
Maybe

~~~
Don't know where to start? 
Please see the current list of [issues at GitHub][issues] for ideas for contributing to this repository.

[issues]: https://github.com/swcarpentry/shell-novice/pull/68/files
~~~